pkg/binfmt/Makefile

```diff
@@ -13,3 +13,11 @@ tag: $(DEPS)
 push: tag
 	DOCKER_CONTENT_TRUST=1 docker pull $(ORG)/$(IMAGE):$(HASH) || \
 	DOCKER_CONTENT_TRUST=1 docker push $(ORG)/$(IMAGE):$(HASH)
+
+check:
+	@if curl -sSL https://registry.hub.docker.com/v2/repositories/$(ORG)/$(IMAGE)/tags/$(HASH) | grep -q 'Not found'; then\
```

Comment:
for small images `pull` is ok, but for larger images it might take quite a while. But i agree re `curl`. Maybe we can add something to the `moby` to do a quick check (including signing)?